package.js

```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: "jazeee:spiderable-longer-timeout",
-  summary: "Extended spiderable package with caching, longer timeout, no stdin issues, and a publish flag.",
-  version: "1.2.2",
+  summary: "Extended spiderable package with support of SSL, caching, longer timeout, no stdin issues, and a publish flag.",
```

Comment:
I will have to look at this. I think Meteor limits the package summary to 80 characters.
We'll find out when I publish...